.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on: push
+
+jobs:
+  review:
+    name: Run static analysis of source code
+    runs-on: ubuntu-latest
+
+    container:
+      image: crystallang/crystal:latest
```

Comment:
I'd recommend using a matrix instead of only `latest` here. Our `shard.yml` specifies that anything greater than or equal to `0.32.0` will work, so we should be testing against all of them. You can see an example on a shard I maintain here:

https://github.com/stephendolan/decorator/blob/main/.github/workflows/shard.yml#L16-L27

Reply:
I really doubt this thing runs still on those old versions, I probably just never updated the min version. I suppose this is a good way to find out though and get that updated